hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/addressbook/AddressBookServiceImplTest.java

```diff
@@ -962,39 +966,77 @@ void getNodesEmptyNodeStake() {
+        }
+        mirrorProperties.setConsensusMode(mode);
 
         assertThat(addressBookService.getNodes())
                 .hasSize(TEST_INITIAL_ADDRESS_BOOK_NODE_COUNT)
-                .allSatisfy(c -> assertThat(c).returns(totalStake, ConsensusNode::getTotalStake))
+                .allSatisfy(c -> assertThat(c).returns(expectedTotalStake, ConsensusNode::getTotalStake))
                 .allSatisfy(c -> assertThat(c.getPublicKey()).isNotNull())
-                .allMatch(c -> c.getStake() > 1L)
+                .allMatch(c -> c.getStake() == expectedNodeStake)
                 .extracting(ConsensusNode::getNodeId)
                 .containsExactly(0L, 1L, 2L, 3L);
     }
 
-    @Test
-    void getNodesWithNodeStakeCountMoreThanAddressBook() {
+    @CsvSource(textBlock = """
+    STAKE, 5, 10
+    STAKE_IN_ADDRESS_BOOK, 5, 8
+    """)
+    @ParameterizedTest
+    void getNodesWithNodeStateCountMoreThanAddressBook(ConsensusMode mode, int nodeCount, long expectedTotalStake) {
         long timestamp = domainBuilder.timestamp();
-        domainBuilder.nodeStake().customize(n -> n.consensusTimestamp(timestamp).nodeId(0).stake(0L)).persist();
-        domainBuilder.nodeStake().customize(n -> n.consensusTimestamp(timestamp).nodeId(1).stake(0L)).persist();
-        domainBuilder.nodeStake().customize(n -> n.consensusTimestamp(timestamp).nodeId(2).stake(0L)).persist();
-        domainBuilder.nodeStake().customize(n -> n.consensusTimestamp(timestamp).nodeId(3).stake(0L)).persist();
-        domainBuilder.nodeStake().customize(n -> n.consensusTimestamp(timestamp).nodeId(4).stake(0L)).persist();
+        var nodeId = new AtomicInteger(0);
+        for (int i = 0; i < nodeCount; i++) {
+          var nodeStake = domainBuilder.nodeStake().customize(n -> n.consensusTimestamp(timestamp)
+              .nodeId(nodeId.getAndIncrement()).stake(2L)).persist();
+        }
+        mirrorProperties.setConsensusMode(mode);
 
         assertThat(addressBookService.getNodes())
                 .hasSize(TEST_INITIAL_ADDRESS_BOOK_NODE_COUNT)
-                .allMatch(c -> c.getStake() == 1L)
-                .allMatch(c -> c.getTotalStake() == 5L)
+                .allSatisfy(c -> assertThat(c.getPublicKey()).isNotNull())
+                .allMatch(c -> c.getStake() == 2L)
+                .allMatch(c -> c.getTotalStake() == expectedTotalStake)
                 .allMatch(c -> c.getNodeAccountId().getEntityNum() - 3 == c.getNodeId())
-                .allSatisfy(c -> assertThat(c.getPublicKey()).isNotNull());
+                .extracting(ConsensusNode::getNodeId)
+                .containsExactly(0L, 1L, 2L, 3L);
+    }
+
+    @CsvSource(textBlock = """
+    STAKE_IN_ADDRESS_BOOK, 40000, 10000
+    STAKE, 60000, 10000
+    EQUAL, 6, 1
+    """)
+    @ParameterizedTest
+    void getNodesWithNodesNotInAddressBook(ConsensusMode mode, long expectedTotalStake, long expectedNodeStake) {
```

Comment:
Swap the order of the params to be consistent with the order from the other tests. Also can revert to the previous method name for clarity.
```suggestion
    void getNodesWithNodeStakeCountMoreThanAddressBook(ConsensusMode mode, long expectedNodeStake, long expectedTotalStake) {
```

Reply:
Done/